packages/verified-fetch/src/utils/walk-path.ts

```diff
@@ -24,11 +25,15 @@ export async function walkPath (blockstore: ReadableStorage, path: string, optio
   }
 
   if (terminalElement == null) {
-    throw new Error('No terminal element found')
+    throw new CodeError('No terminal element found', 'NO_TERMINAL_ELEMENT')
   }
 
   return {
     ipfsRoots,
     terminalElement
   }
 }
+
+export function objectNodeGuard (node: UnixFSEntry): node is ObjectNode {
```

Comment:
`objectNodeGuard` is a noun, which is normally a class.

```suggestion
export function isObjectNode (node: UnixFSEntry): node is ObjectNode {
```